src/main/java/cnlabs/CustomerWebsite/Models/Book.java

```diff
@@ -0,0 +1,35 @@
+package cnlabs.CustomerWebsite.Models;
+
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+
+@Entity
+@Table(name = "books")
+@Getter
+@Setter
+@RequiredArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id", nullable = false)
+    private Long id;
+
+    @Column(nullable = false)
+    @NotBlank(message = "Title must not be blank.")
+    private String title;
+
+    @NotBlank(message = "Author must not be blank.")
+    private String author;
+
+    @OneToOne(mappedBy = "book")
+    private Customer customer;
+
+    @Override
+    public String toString() {
+        return title + " by " + author;
+    }
+}
```

Comment:
I would recommend always adding "guarding" empty line at the end. 
For Java, it is not a problem, but so some Linux scritps it could be an issue.

Reply:
You mean just a blank line 36?